CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.1.0]
+### Added
+- Fix: saving serialized keyring for which corresponding keyring class is not present ([#169](https://github.com/MetaMask/KeyringController/pull/169))
```

Comment:
Not sure that fix is the best description for this change. Maybe this a change entry in need of re-wording instead.

The context here is that the keyring controller we use today only supports adding/restoring/deserializing _known_ keyring types. If you pass it a keyring it doesn't recognize, it crashes. This is not really a bug, it's intentional.

Jyoti has added support for deserializing vaults that include unsupported keyrings. Now we can safely deserialize vaults that include unsupported keyrings, and we keep the unsupported data in the vault in case it becomes supported again in the future (or in case support needs to walk someone through the steps to manually extract it).

Reply:
Ah I see. I misunderstood.

I'm not sure how best to reword this right now but I can revisit this tomorrow and come up with a suggestion.